homeassistant/components/websocket_api/connection.py

```diff
@@ -37,6 +38,16 @@ def __init__(
         self.subscriptions: dict[Hashable, Callable[[], Any]] = {}
         self.last_id = 0
 
+        def revoke_callback() -> None:
+            self._token_revoked = True
+
+        self._token_revoked = False
+        self._unregister_revoke_callback = (
+            self.hass.auth.async_register_revoke_token_callback(
```

Comment:
The responsibility for registering this callback is better fit in `AuthPhase` (`websocket_api/auth.py`)

Reply:
Updated this here d159ee4